src/stretchy.js

```diff
@@ -342,50 +342,43 @@ const encloseSpan = function(
 const ruleSpan = function(className: string, lineThickness: number,
     options: Options): domTree.span {
 
-    // Get a span with an SVG line that fills the middle fifth of the span.
+    // Get a span with an SVG path that fills the middle fifth of the span.
     // We're using an extra wide span so Chrome won't round it down to zero.
 
-    const lines = [];
+    let path;
     let svgNode;
-    if (className === "vertical-separator") {
-        // Apply 2 brush strokes for sharper edges on low-res screens.
-        for (let i = 0; i < 2; i++) {
-            lines.push(new domTree.lineNode({
-                "x1": "5",
-                "y1": "0",
-                "x2": "5",
-                "y2": "10",
-                "stroke-width": "2",
-            }));
-        }
+    let parentClass = "stretchy";  // default
 
-        svgNode = new domTree.svgNode(lines, {
-            "width": "0.25em",
-            "height": "100%",
-            "viewBox": "0 0 10 10",
-            "preserveAspectRatio": "none",
+    if (className === "vertical-separator") {
+        path = new domTree.pathNode("vertSeparator");
```

Comment:
Cool.  Thanks for updating vertical separators to be consistent with the horizontal ones.